run.go

```diff
@@ -38,37 +34,22 @@ func NewRun[OC any, JC any](name string, oc OC) *Run[OC, JC] {
 func (r *Run[OC, JC]) Init() {
 	r.m.Lock()
 	defer r.m.Unlock()
-	r.lockedJobsById = map[string]string{}
-	r.lockedJobsStateCount = map[string]int{}
```

Comment:
Overall yes I think this is a good move... the locking is not really data we care about to serialize which run is all about (serializable state), it belongs in processor or StateExec which is much more about the in flight processing status.